client/components/RatingsAndReviews/RatingsAndReviews.jsx

```diff
@@ -8,14 +8,18 @@ import { makeStyles } from '@material-ui/core/styles';
 const useStyles = makeStyles((theme) => ({
   root: {
     flexGrow: 1,
+  },
+  header: {
+    marginBottom: 35
   }
 }));
 
 const RatingsAndReviews = (props) => {
   const classes = useStyles();
   return (
     <>
-    <Typography variant="subtitle1">RATINGS & REVIEWS</Typography>
+    <Typography variant="subtitle1" class={classes.header}>RATINGS & REVIEWS</Typography>
```

Comment:
Getting an error regarding line 21: class I think should be className?  It doesn't appear to be affecting the functionality, but just to get rid of the warning.

Reply:
Addressed!

Reply:
Thanks!